tests/Meilisearch.Tests/IndexFixture.cs

```diff
@@ -197,9 +197,9 @@ public async Task<Index> SetUpIndexForNestedSearch(string indexUid)
         public async Task DeleteAllIndexes()
         {
             var indexes = await DefaultClient.GetAllIndexesAsync();
-            foreach (var index in indexes)
+            foreach (var index in indexes.Results)
```

Comment:
Are you returning an `Index` from this list, don't you?

So I think you don't need the `DefaultClient.Index(index.Uid)` you can just `index.DeleteAsync();`

Reply:
It's because line 143 is not working properly so it didn't work.